### PR TITLE
Bwa plus tools

### PR DESCRIPTION
**Pull request by KateSakharova**
- bwa-mem2 + metabat2 + concoct
- bwa-mem2 + eukcc (in progress)

I tried to build using conda but bumped into multiple problems:
- Eukcc has issues rinning `COPY --from=miniconda / /`
- concoct can be installed with conda but raises problems during run

### CODE REVIEW

File: bwa_eukcc/2.2.1_2.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    && python3 setup.py sdist bdist_wheel \
+    && pip3 install -e .
+
+RUN mkdir /data
+WORKDIR /data
+
+# for singularity compatibility
+ENV LC_ALL=C
+
+COPY ggp_binlinks.py /usr/local/bin/
```
Comment (mberacochea) [inline bwa_eukcc/2.2.1_2.0/Dockerfile:48]: I think we should remove this one, and it should only live in the GGP pipeline

Comment (mberacochea) [reply bwa_eukcc/2.2.1_2.0/Dockerfile:48]: Nextflow will load the script automatically from the bin directory

File: bwa_metabat_concoct/2.2.1_2.16_1.1.0/Dockerfile
Comment (mberacochea) [inline bwa_metabat_concoct/2.2.1_2.16_1.1.0/Dockerfile:1]: How big is this container? If it's too big we could use multi-stage to slim it down

**Review by mberacochea — COMMENTED** [review]
Thanks Kate, I left 2 comments

File: bwa_metabat_concoct/2.2.1_2.16_1.1.0/Dockerfile
Comment (KateSakharova) [reply bwa_metabat_concoct/2.2.1_2.16_1.1.0/Dockerfile:1]: I tried multi build but I had issues with --copy-from one image to another. You can try... I gave up after a couple of days and ChatGPT told me I'm dummy :D
https://quay.io/repository/microbiome-informatics/bwa_metabat_concoct?tab=tags
if quays shows size of image then 600Mb.

Comment (mberacochea) [reply bwa_metabat_concoct/2.2.1_2.16_1.1.0/Dockerfile:1]: 600Mb.. that is just fine

**Pull request merged**